includes/class-wcqu-product-meta-box.php

```diff
@@ -18,39 +18,39 @@ public function meta_box_create() {
 		global $post, $woocommerce;
 
 		if ( $post->post_type == 'product' ) {
-			
-			$product = get_product( $post->ID );
+
+			$product = wc_get_product( $post->ID );
 			$unsupported_product_types = array( 'external', 'grouped' );
 
-			if ( ! in_array( $product->product_type, $unsupported_product_types ) ) {
-						
+			if ( ! in_array( $product->get_type(), $unsupported_product_types ) ) {
+
 				add_meta_box(
-					'wpbo_product_info', 
-					__('Product Quantity Rules', 'woocommerce'), 
-					array( $this, 'product_meta_box_content' ), 
-					'product', 
-					'normal', 
-					'high' 
+					'wpbo_product_info',
+					__('Product Quantity Rules', 'woocommerce'),
+					array( $this, 'product_meta_box_content' ),
+					'product',
+					'normal',
+					'low'
```

Comment:
Out of curiosity: what does this do?

Reply:
It adds a new wordpress meta box to the woocommerce product page.

This one here:
![image](https://user-images.githubusercontent.com/42588836/134867309-ef605b32-a9f2-4c2d-80f8-c7e012eeed39.png)

The priority change from `high` to `low` affects the render position on the screen. Now the plugin meta box will be at the bottom of the screen and not on top.